spring-cloud-gateway-core/src/main/java/org/springframework/cloud/gateway/route/Route.java

```diff
@@ -177,6 +197,16 @@ public B uri(URI uri) {
 			return getThis();
 		}
 
+		public B putMetadata(Map<String, Object> options) {
```

Comment:
to be consistent, I think this method should be called `metadata`

Reply:
done